chef/site-cookbooks/wca/templates/worldcubeassociation.org.conf.erb

```diff
@@ -172,6 +172,11 @@ server {
     }
   }
 
+  # Redirect /admin/change_person.php to /admin/change_person
```

Comment:
Nice! When deploying this, we're going to need to re-run chef. If you don't know how to do this, please reach out to me when the time comes =)